### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by bboule**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -50,14 +20,55 @@ To test out the beta version for yourself,
...
+Note that only newly created continuous aggregates will have this real-time query behavior, although it can be enabled on existing continuous aggregates 
+with a configuration setting as follows:
+
+ALTER VIEW continuous_view_name SET (timescaledb.materialized_only=false);
+
+This release also moves several data management lifecycle features to the Community version of TimescaleDB (from Enterprise), 
+including data reordering and data retention policies.
+
+**Major Features**
+*	#1456 Add support for PostgreSQL 12
```
Comment (svenklemm) [inline release-notes.md:48]: can you link #1807 here instead because thats the PR actually implementing it

Comment (bboule) [reply release-notes.md]: I have made that update

File: release-notes.md [hunk 2]
```diff
@@ -5,43 +5,19 @@ Future Plans section. Interested in learning more about
 what's already available? Jump down below to see what's
 been released.
 
+## PostgreSQL Version deprecation Notice
+
+Please note that with the release of Timescale 1.7 we are deprecating support for 
+PostgreSQL 9.x and PostgreSQL 10.x  The current plan is to remove support for these versions of PostgreSQL
```
Comment (svenklemm) [inline release-notes.md:11]: 9.6.x

File: release-notes.md [hunk 3]
```diff
@@ -5,43 +5,19 @@ Future Plans section. Interested in learning more about
 what's already available? Jump down below to see what's
 been released.
 
+## PostgreSQL Version deprecation Notice
```
Comment (svenklemm) [inline release-notes.md:8]: this looks slightly weird with 3 out of weird capitalized

File: release-notes.md [hunk 4]
```diff
@@ -5,43 +5,19 @@ Future Plans section. Interested in learning more about
 what's already available? Jump down below to see what's
 been released.
 
+## PostgreSQL version deprecation notice
+
+Please note that with the release of Timescale 1.7 we are deprecating support for 
+PostgreSQL 9.6.x and PostgreSQL 10.x  The current plan is to remove support for these versions of PostgreSQL
+as part of the Timescale 2.0 release later this year.
```
Comment (mfreed) [inline release-notes.md:12]:
```suggestion
Please note that with the release of Timescale 1.7,
we are deprecating support for PostgreSQL 9.6.x and 10.x.
The current plan is that the Timescale 2.0 release later this year will
only support PostgreSQL major versions 11.x, 12.x, or newer.
```

File: release-notes.md [hunk 5]
```diff
@@ -50,14 +20,65 @@ To test out the beta version for yourself,
 follow these [instructions](https://docs.timescale.com/clustering/getting-started/scaling-out).
```
Comment (mfreed) [inline release-notes.md:20]: What do you think about pointing to blog instead?  The connected docs don't really give a picture of the architecture & why you'd use -- a bit low level?

Comment (mfreed) [reply release-notes.md:20]: Namely, both of the articles, maybe as a bulleted list or something?

- https://blog.timescale.com/blog/building-a-distributed-time-series-database-on-postgresql/
- https://blog.timescale.com/blog/achieving-optimal-query-performance-with-a-distributed-time-series-database-on-postgresql/

Comment (bboule) [reply release-notes.md:20]: @mfreed are you ok publishing for now as is and let me update this next week (I agree with the suggestion I am just a little short on time but will do it over the weekend) but was hoping to push docs today just so they are up there and match the fact that the release is out  OK by you?

**Pull request merged**